hack/verify-generated-codes.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubeflow Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+cd "$(dirname "$0")/.."
+
+git diff --exit-code -- 'pkg/apis' 'hack/gen-python-sdk' 'sdk/python' 'pkg/mock' > /dev/null || \
```

Comment:
@andreyvelich Are there any other directories that need to be checked?

Reply:
@tenzen-y What do you think about verifying the whole code base on the changes
(e.g. `git diff --exit-code -- . `)?
For example, tomorrow we are going to introduce script check to automatically sync `operator` manifests with our manifests and we ask users to run this script before submitting PR.

Reply:
It makes sense.
I will change so that developer can check all files.